MetafileImporter/Logic/vtkSlicerMetafileImporterLogic.cxx

```diff
@@ -187,6 +187,41 @@ void UpdateTransformNodeFromString(vtkMRMLLinearTransformNode* transformNode, co
 }
 
 
+void UpdateStringFromTransformNode(std::string& str, vtkMRMLLinearTransformNode* transformNode)
```

Comment:
> add analogous method

Makes sense.

I guess the following code could be factored out: https://github.com/Slicer/Slicer/blob/029ee22139b85916b0b5b07be42c081e77dfd6e4/Modules/Loadable/Transforms/qSlicerTransformsModuleWidget.cxx#L394-L465

```
std::string toString(vtkMatrix4x4* mat, const std::string rowDelimiter = " ") const; 
bool fromString(vtkMatrix4x4* mat, const std::string& str); # kwsys regex could be used ?
```

Cc: @lassoan